examples/notebooks/custom-metadata/custom_metadata.ipynb

```diff
@@ -141,11 +141,13 @@
         "\n",
         "step = Step(name=\"Initial step\", test_result_id=test_result_id)\n",
         "step_id = data_store_client.create_step(step)\n",
-        "published_measurement = data_store_client.publish_measurement(\n",
+        "published_measurement_id = data_store_client.publish_measurement(\n",
         "    name=\"scope reading\",\n",
         "    value=waveform,\n",
         "    step_id=step_id,\n",
-        ")"
+        ")\n",
+        "\n",
+        "published_measurement = data_store_client.get_measurement(published_measurement_id)"
```

Comment:
Unless we are getting data or other fields, we should not need to call get_measurement.